x-pack/plugins/infra/server/lib/adapters/log_entries/kibana_log_entries_adapter.ts

```diff
@@ -8,10 +8,7 @@
 
 import { timeMilliseconds } from 'd3-time';
 import * as runtimeTypes from 'io-ts';
-import first from 'lodash/fp/first';
-import get from 'lodash/fp/get';
-import has from 'lodash/fp/has';
-import zip from 'lodash/fp/zip';
+import _ from 'lodash';
```

Comment:
Could also do

```suggestion
import { first, get, has, zip } from 'lodash';
```

to avoid importing unnecessary functions and be consistent with other files in this PR